network/nebula/README.md

```diff
@@ -0,0 +1,49 @@
+# Nebula
+
+https://github.com/slackhq/nebula
+
+## Installation
+
+See [Installing Extensions](https://github.com/siderolabs/extensions#installing-extensions).
+
+## Usage
+
+Configure the extension via `ExtensionServiceConfig` document. You can add any nebula related configuration and these will be executed at Nebula startup.
+
+```yaml
+---
+apiVersion: v1alpha1
+kind: ExtensionServiceConfig
+name: nebula
+configFiles:
+  - content: |
+        pki:
```

Comment:
is `pki` an empy key or whether `ca`, `cert` and `key` should be under `pki`

Reply:
`ca`, `cert`, and `key` are supposed to be under `pki`

Reply:
okay, I'll fix the example, here pki and ca are all under same indentation